### PR TITLE
drop bionic support

### PR DESCRIPTION
**Pull request by rgildein**
The functional tests are failing for Bionic series, and since we do not need to support it anymore.

### CODE REVIEW

**Review by sudeephb — APPROVED** [review]

**Review by agileshaw — CHANGES_REQUESTED** [review]
Now that we no longer support bionic, we should un-pin `ops` in `requirements.txt` and update README (which contains an example of bionic deployment).

**Review by aieri — APPROVED** [review]

**Comment by rgildein** [comment]
> Now that we no longer support bionic, we should un-pin `ops` in `requirements.txt` and update README (which contains an example of bionic deployment).

Thanks, I removed any mention of Bionic (from examples, tests and default values).

File: tests/functional/test_charm.py [hunk 1]
```diff
@@ -437,17 +437,17 @@ async def test_outdated_packages_distro_changed(
         count = int(results.get("count"))
         assert count == 0
 
-        # Let's switch back to bionic and create a snapshot before switching to
+        # Let's switch back to Focal and create a snapshot before switching to
         # focal.
```
Comment (agileshaw) [inline tests/functional/test_charm.py]:
```suggestion
        # Let's switch back to Focal and create a snapshot before switching to
        # jammy.
```

**Review by agileshaw — COMMENTED** [review]
Just one small change suggested inline, otherwise looks good.

**Review by agileshaw — APPROVED** [review]
LGTM, thanks

**Pull request merged**